app/assets/javascripts/discourse/app/initializers/subscribe-user-changes.js

```diff
@@ -7,8 +7,10 @@ export default {
 
     if (user) {
       const bus = container.lookup("message-bus:main");
+      const appEvents = container.lookup("service:app-events");
       bus.subscribe("/user", (data) => {
         user.setProperties(data);
+        appEvents.trigger("current-user:updated", data);
```

Comment:
This is great, but I do worry that people will expect this to be called any time the user changes and there are many other places where we update it that wouldn't be covered by this.

I suggest something like `current-user:updated-via-bus`